### ISSUE TITLE
BUG: CSV branch does not successfully write CSV - character maps to <undefined>

### ISSUE THREAD

**Issue by Pizzanomicon** [hanzhi713/image-collage-maker#14]
Upon clicking "Save tile info" in the GUI, a window pops up:

Error

```
Traceback (most recent call last):
File "C:\Resources\image-collage-maker-csv\gui.py", line 679, in save_tile_info
f.write(result_tile_info)
File "C:\ Program Files\WindowsApps\PythonSoftware Foundation.Python.3.10_3.10.1264.0_x64__qbz5n2kfra8p0\lib\encodings\cp1252.py".
line 19, in encode
return
codecs.charmap_encode(input, self.errors, encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters
in position 1839074-1839079: character maps to <undefined>
```

Could this be an illegal character in my filenames?  The mosaic graphic generated and saved just fine.

**Comment by Pizzanomicon** [comment]
I believe this might be the fix? https://stackoverflow.com/questions/27092833/unicodeencodeerror-charmap-codec-cant-encode-characters

**Comment by hanzhi713** [comment]
Should be fixed by 7e808b97c199558446caebb7c9f4566e6182d54b
